pkg/scheduler/queue/block_queue.go

```diff
@@ -380,15 +380,15 @@ func (p *BlockQueue) AddUnschedulableIfNotPresent(unitInfo *framework.QueuedUnit
 	queue, prevUnitInfo, exists := p.findAndRemoveUnitFromQueue(unitInfo.UnitKey)
```

Comment:
rename `prevUnitInfo` to `unitInfoInSchedulingQueue`,  `prevUnitInfo` is misleading

Reply:
good idea.